Code/Tools/AssetProcessor/native/utilities/PlatformConfiguration.cpp

```diff
@@ -2124,7 +2124,7 @@ namespace AssetProcessor
                 addedEntries = true;
             }
         }
-        return true;
+        return addedEntries;
```

Comment:
I assume that the function was always supposed to return this value. However, this is more than an unused variable change because the function can now return false. I would confirm with whoever maintains this system that changing the return value won't change behavior unexpectedly.

Reply:
At least in o3de, the one use of this function ignores the return value, and this is pretty deep in AP implementation. I suspect this change won't have any practical impact and seems likely to be "more correct", since there's no other good reason for addedEntries to exist.

Reply:
> 

at that point just do away with addedEntries.

Reply:
It pretty common to make a function return a boolean signature to indicate if the function had any affect, but in this case, it doesnt look like any caller of this function actually cares about whether something was added or not. 
Also of note, there is an existing PR ( https://github.com/o3de/o3de/pull/7333 ) that has this change as well as a lot more unused variable clean up